CMakeLists.txt

```
@@ -258,6 +258,12 @@ target_sources(${QUOTIENT_LIB_NAME} PUBLIC FILE_SET HEADERS BASE_DIRS .
         libquotientemojis.qrc
 )
 
+include(ECMQmlModule)
+include(KDEInstallDirs)
+
+ecm_add_qml_module(${QUOTIENT_LIB_NAME} URI com.github.quotient_im.libquotient GENERATE_PLUGIN_SOURCE)
```

```suggestion
ecm_add_qml_module(${QUOTIENT_LIB_NAME} URI io.github.quotient_im.libquotient GENERATE_PLUGIN_SOURCE)
```
To stay aligned with the `io.github.org.project` form required by Flathub.